styles.css

```diff
@@ -1,504 +1,1051 @@
+    color: #172b4d;
+  }
 
-.connect {
-  width: 177px;
-  height: 14px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 22px;
-  line-height: 24px;
-  padding-left: 26px;
-  color: #6070ff;
-  text-decoration: none;
-}
+  .paragraph {
+    width: 327px;
+    height: 128px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    padding-left: 26px;
+    margin-top: 0;
+  }
 
-.social {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  width: 170.17px;
-  height: 20px;
-  list-style: none;
-  padding-left: 28px;
-  position: relative;
-  bottom: 22px;
-}
+  .connect {
+    width: 177px;
+    height: 14px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 24px;
+    padding-left: 26px;
+    color: #6070ff;
+    text-decoration: none;
+  }
 
-.meta {
-  margin-top: 3px;
-}
+  .social {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 170.17px;
+    height: 20px;
+    list-style: none;
+    padding-left: 28px;
+    position: relative;
+    bottom: 22px;
+  }
 
-/* work section */
+  .meta {
+    margin-top: 3px;
+  }
 
-.works {
-  display: grid;
-  grid-template-rows: repeat(3, 1fr);
-  row-gap: 60px;
-  align-items: center;
-  width: 337px;
-}
+  /* work section */
 
-.cards {
-  display: grid;
-  grid-template-rows: 1fr 1fr;
-  align-items: center;
-  justify-content: center;
-  background: #fff;
-  padding-bottom: 25px;
-  border-radius: 20px;
-}
+  .works {
+    display: grid;
+    grid-template-rows: repeat(3, 1fr);
+    row-gap: 60px;
+    align-items: center;
+    width: 337px;
+  }
 
-.card-img {
-  width: 295px;
-  height: 220px;
-  border-radius: 8px;
-}
+  .cards {
+    display: grid;
+    grid-template-rows: 1fr 1fr;
+    align-items: center;
+    justify-content: center;
+    background: #fff;
+    padding-bottom: 25px;
+    border-radius: 20px;
+  }
 
-.block2 {
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
-  gap: 12px;
-  width: 295px;
-  height: 288px;
-  position: relative;
-  bottom: 22px;
-}
+  .card-img {
+    width: 295px;
+    height: 220px;
+    border-radius: 8px;
+  }
 
-.box1 {
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
-  width: 295px;
-  height: 80px;
-}
+  .block2 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 12px;
+    width: 295px;
+    height: 288px;
+    position: relative;
+    bottom: 22px;
+  }
 
-.title1 {
-  color: #172b4d;
-  width: 295px;
-  height: 36px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
-}
+  .box1 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 295px;
+    height: 80px;
+  }
```

Comment:
- [ ] Please ensure that "CANOPY" follows the design guidelines and does not have any list decorations. This is important to maintain consistency and coherence with the overall design aesthetic.

**Your design**

![Screenshot from 2023-03-31 18-26-08](https://user-images.githubusercontent.com/105117832/229249850-942364f2-f86f-4aeb-82d6-de4bf719b7ab.png)

**Figma**

![Screenshot from 2023-03-31 18-26-18](https://user-images.githubusercontent.com/105117832/229249864-f77e2662-1520-4d19-9956-d32ea359fc2e.png)

- [ ] You have done a remarkable job in designing the buttons within your project :+1:. However, to align with the Figma template, it is crucial that the buttons have the same background color as the template. Therefore, I kindly request you to make the necessary changes to maintain consistency with the design guidelines.

**Your design**

![Screenshot from 2023-03-31 18-30-42](https://user-images.githubusercontent.com/105117832/229250082-11970256-3454-4a14-a739-0c18094b3c74.png)

**Figma**

![Screenshot from 2023-03-31 18-30-55](https://user-images.githubusercontent.com/105117832/229250095-30a3ad80-9564-447a-b4e9-8abfe7f7a947.png)